src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -217,18 +217,18 @@ private function addFragmentsSection(ArrayNodeDefinition $rootNode)
         ;
     }
 
-    private function addProfilerSection(ArrayNodeDefinition $rootNode)
+    private function addInspectorSection(ArrayNodeDefinition $rootNode)
     {
         $rootNode
             ->children()
                 ->arrayNode('profiler')
```

Comment:
This configuration key should be renamed to inspector, shouldn't it?

Reply:
That would be a big BC break.

Reply:
well, we can have a BC layer for that. That's the big benefit of the semantic configuration system.